assets/src/edit-story/output/page.js

```diff
@@ -78,7 +78,11 @@ function OutputPage({ page, autoAdvance, defaultPageDuration }) {
         <StoryAnimation.AMPAnimations />
 
         {backgroundElement && (
-          <amp-story-grid-layer template="vertical" aspect-ratio={ASPECT_RATIO}>
+          <amp-story-grid-layer
+            id="amp-story-grid-layer-bg"
```

Comment:
`id` attributes should be unique in a given HTML document. This element will exists dozens of times though, so it should really use `class`.

Reply:
Also, maybe drop the `amp-story-` prefix for our custom class names.

Reply:
Sure, will try.

Reply:
Resolved by using `body.amp-mode-mouse amp-story-grid-layer`